core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -2742,31 +2734,19 @@ public function overrideAll()
 
 				try
 				{
+					$this->preloadCurrentRecords($ids);
+
 					foreach ($ids as $id)
 					{
-						try
-						{
-							$this->denyAccessUnlessGranted(ContaoCorePermissions::DC_ACTION_EDIT, new DataContainerSubject($this->strTable, $id));
-						}
-						catch (AccessDeniedException)
-						{
-							continue;
-						}
+						$currentRecord = $this->getCurrentRecord($id);
+
+						$this->denyAccessUnlessGranted(ContaoCorePermissions::DC_PREFIX . $this->strTable, new ReadAction($this->strTable, $currentRecord));
```

Comment:
I don't think `overrideAll` needs a `ReadAction` check, because no data is read or displayed to the user. Also, if we would add one, the try-catch-continue would be missing.